reference/clinic.v1.yaml

```diff
@@ -253,22 +253,46 @@ paths:
           in: query
           name: period
           description: Time Period to display
-        - schema:
+          required: true
+        - name: tags
+          description: Comma-separated list of patient tag IDs
+          schema:
             type: array
             items:
               type: string
-              format: '^[a-f0-9]{24}$'
+              pattern: '^[a-f0-9]{24}$'
+            minItems: 1
           in: query
-          name: tags
-          description: Comma-separated list of patient tag IDs
           style: form
           explode: false
-        - schema:
+          required: true
+        - name: lastDataCutoff
+          description: Inclusive minimum of date of last data from a patient.
```

Comment:
Is there a limit to how far back this can go? That might be good to call out in the description.

Reply:
No limit.